app.js

```diff
@@ -56,6 +57,102 @@ app.get(`/${config.admin_route}`, (req, res) => {
   })(req, res)
 })
 
+/* API Routes */
+
+// Users
+// gets and sends back first 20 users in db, starting with admins (sorting can be improved)
+app.get('/api/users/', passport.authenticate('loggedIn', { session: false }),
+  (req, res, next) => {
+    userModel.find({}, null, { limit: 20 }).sort({ isAdmin: -1 }).exec((err, docs) => {
+      if (!err) {
+        // remove password fields from docs
+        const responseDocs = docs.map(doc => {
+          return {
```

Comment:
You can remove the password field by doing:
```js
delete doc['password']
```

Reply:
fair enough

Reply:
for some reason `delete doc.password` or `delete doc['password']` didn't work but `doc.password = undefined` does

Reply:
@zeim839 I'm going to add some more sanity checking for the put method before I push